deploy/vagrant/modules/javascript/templates/run-jscover-qunit.js.erb

```diff
@@ -133,9 +133,6 @@ page.open(system.args[1], function(status){
                 } catch (e) { }
                 return 10000;
             });
-            page.evaluate(function(){
-                jscoverage_report('js-coverage');
-            });
```

Comment:
This hasn't run successfully in a long time, so i deleted it while i was here to get rid of a spurious error.